doc/LaunchingVitess.md

```diff
@@ -435,40 +404,39 @@ VTTablet has a large number of command line options. Some important ones will be
-
 * **queryserver-config-stream-pool-size**: This value is relevant only if you plan to run streaming queries against the database. It’s recommended that you use rdonly instances for such streaming queries. This value depends on how many simultaneous streaming queries you plan to run. Typical values are in the low 100s.
-
 * **queryserver-config-transaction-cap**: This value should be set to how many concurrent transactions you wish to allow. This should be a function of transaction QPS and transaction length. Typical values are in the low 100s.
-
 * **queryserver-config-query-timeout**: This value should be set to the upper limit you’re willing to allow a query to run before it’s deemed too expensive or detrimental to the rest of the system. VTTablet will kill any query that exceeds this timeout. This value is usually around 15-30s.
-
 * **queryserver-config-transaction-timeout**: This value is meant to protect the situation where a client has crashed without completing a transaction. Typical value for this timeout is 30s.
-
 * **queryserver-config-max-result-size**: This parameter prevents the OLTP application from accidentally requesting too many rows. If the result exceeds the specified number of rows, VTTablet returns an error. The default value is 10,000.
 
 ### DB config parameters
 
-VTTablet requires multiple user credentials to perform its tasks. Any requests coming from the app will only use the app credentials. VTTablet is required to run on the same machine as MySQL. Therefore, it’s most beneficial to use the more efficient unix socket connections.
+VTTablet requires multiple user credentials to perform its tasks. Since it's required to run on the same machine as MySQL, it’s most beneficial to use the more efficient unix socket connections.
 
+**app** credentials are for serving app queries:
 * **db-config-app-unixsocket**: MySQL socket name to connect to.
-
 * **db-config-app-uname**: App username.
-
 * **db-config-app-pass**: Password for the app username. If you need a more secure way of managing and supplying passwords, VTTablet does allow you to plug into a "password server" that can securely supply and refresh usernames and passwords. Please contact the Vitess team for help if you’d like to write such a custom plugin.
-
 * **db-config-app-charset**: The only supported character set is utf8. Vitess still works with latin1, but it’s getting deprecated.
 
-For other credentials, replace the "app" part of the command line argument with “dba”, “repl”, or “filtered”.
-
-Different users we have:
+**dba** credentials will be used for housekeeping work like loading the schema or killing runaway queries:
+* **db-config-dba-unixsocket**
+* **db-config-dba-uname**
+* **db-config-dba-pass**
+* **db-config-dba-charset**
 
-* App: used to fulfill app requests.
+**repl** credentials are for managing replication:
+* **db-config-repl-unixsocket**
+* **db-config-repl-uname**
+* **db-config-repl-pass**
+* **db-config-repl-charset**
```

Comment:
You also should add:
db-config-repl- followed by the four:
flags
ssl-ca
ssl-cert
ssl-key

they are used for encrypted replication. flags should be 2048 in that case.